packages/blade/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
plans to consume these form `universe-doctor` in future?

Reply:
no plans yet